apps/angular/src/app/features/anime/table-view/table-view.component.html

```diff
@@ -0,0 +1,44 @@
+<section class="table-view">
+  <h2 id="anime-catalog" class="mat-title">Anime Catalog</h2>
+  <table *ngIf="animeList$ | async as anime; else load" mat-table [dataSource]="anime"
+    class="mat-elevation-z8 demo-table" aria-label="anime" aria-describedby="catalog-anime">
```

Comment:
Please place each attribute on the new line, if there more than 3 attributes. Because it's more readable and maintainable.
```suggestion
  <table
    *ngIf="animeList$ | async as anime; else load"
    mat-table
    [dataSource]="anime"
    class="mat-elevation-z8 demo-table"
    aria-label="anime"
    aria-describedby="catalog-anime"
  >
```

Reply:
You can use tools like Prettier to do it automatically